menu.php

```diff
@@ -6,12 +6,12 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Menu</title>
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
-	<link rel="stylesheet" type="text/css" href="css\menu.css">
+	<link rel="stylesheet" type="text/css" href="css\main.css">
```

Comment:
this should be `css/main.css` the `\` is a windows only thing and will not translate to most web servers which use a unix based file system